include/netlink/route/link/can.h

```diff
@@ -51,6 +51,19 @@ extern int rtnl_link_can_get_ctrlmode(struct rtnl_link *, uint32_t *);
 extern int rtnl_link_can_set_ctrlmode(struct rtnl_link *, uint32_t);
 extern int rtnl_link_can_unset_ctrlmode(struct rtnl_link *, uint32_t);
 
+extern int rtnl_link_can_get_data_bt_const(struct rtnl_link *,
```

Comment:
there is a getter. Why is there not setter?

Below we see
```
+                 NLA_PUT(msg, IFLA_CAN_DATA_BITTIMING_CONST, sizeof(ci->ci_data_bittiming_const),
+                         &ci->ci_data_bittiming_const);
```
so it seems relevant to also set those parts. Or how is this supposed to work?

Reply:
the name `rtnl_link_can_get_data_bt_const()` seems inconsistent. I think it should be `rtnl_link_can_get_data_bittiming_const()`?

Reply:
I'll rename `rtnl_link_can_get_data_bt_const()` to `rtnl_link_can_get_data_bittiming_const()`.

I'll also add `rtnl_link_can_set_data_bittiming_const()`.